mirrord-layer/src/lib.rs

```diff
@@ -25,6 +25,7 @@ lazy_static! {
     static ref RUNTIME: Runtime = Runtime::new().unwrap();
     static ref SOCKETS: sockets::Sockets = sockets::Sockets::default();
     static ref NEW_CONNECTION_SENDER: Mutex<Option<Sender<i32>>> = Mutex::new(None);
+    static ref UNIX_FDS: Mutex<HashSet<i32>> = Mutex::new(HashSet::new());
```

Comment:
Future thinking, I think this should be `ignored_fds` as we might have other reasons to ignore fds.